aptos-move/aptos-aggregator/src/types.rs

```diff
@@ -7,7 +7,6 @@ use aptos_types::delayed_fields::{
     bytes_and_width_to_derived_string_struct, derived_string_struct_to_bytes_and_length,
     is_derived_string_struct_layout,
 };
-// TODO[agg_v2](cleanup): After aggregators_v2 branch land, consolidate these, instead of using alias here
```

Comment:
this TODO is to remove the "pub" from the line below, and have everyone directly import those from aptos_types

Reply:
I would also wait for delayed value impl PR to land first, I had to move some things to move third_party, so it will be easier to consolidate all imports after

Reply:
I've made the changes Igor mentioned. Will update again after your PR lands.